docker/envoy_mixer.json

```diff
@@ -0,0 +1,44 @@
+{
+  "listeners": [
+   {
+    "address": "tcp://0.0.0.0:15004",
+    "name": "tcp_0.0.0.0_15004",
+    "filters": [
+     {
+      "type": "read",
+      "name": "tcp_proxy",
+      "config": {
+       "stat_prefix": "tcp",
+       "route_config": {
+        "routes": [
+         {
+          "cluster": "in.9091"
+         }
+        ]
+       }
+      }
+     }
+    ],
+    "bind_to_port": true
+   }
+  ],
+  "admin": {
+    "access_log_path": "/dev/stdout",
+    "address": "tcp://127.0.0.1:15000"
+  },
+  "cluster_manager": {
+    "clusters": [
+      {
+        "name": "in.9091",
+        "connect_timeout_ms": 1000,
+        "type": "static",
+        "lb_type": "round_robin",
+        "hosts": [
+          {
+            "url": "tcp://127.0.0.1:9091"
+          }
+        ]
+      }
+    ]
+  }
+}
```

Comment:
please add the statsd and tracing configuration as well. The proxy agent has CLI options to specify the statsd cluster and the zipkin cluster. Statsd config is useful immediately. Tracing will become useful soon after we get tracing for xDS

Reply:
Shriram, we are not using pilot agent to generate the envoy configuration. I could add a static configuration, but it will not work if mesh config is changed. This is why I had pilot-agent generating envoy.json startup file in my previous iteration. 

https://github.com/istio/pilot/pull/1444

Reply:
"proxy agent has CLI options to specify the statsd cluster and the zipkin cluster"
specifically I can't use the CLI options to proxy agent because the files are static. I prefer my previous approach where proxy agent generates the file... but you vetoed it :)

Reply:
Ah got it. Sorry. ignore the comment